src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-700.mdx

```diff
@@ -0,0 +1,26 @@
+---
+subject: Android agent
+releaseDate: '2023-06-08'
+version: 7.0.0
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_7.0.0.zip'
+---
+
+
+## New
+
+* AGP 8 support
```

Comment:
Adds support for Gradle 8.0 and higher
Adds support for the Android Gradle Plugin 8.0 and higher
Adds support for Gradle configuration caching

Reply:
Adds support for JDK 17 when used with Gradle 8